src/styles/blocks/header.scss

```diff
@@ -0,0 +1,70 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+}
+
+.header__logo-img {
+  width: 40px;
+  height: 40px;
+
+  display: block;
+}
+
+.nav {
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  position: relative;
+
+  height: 60px;
+  line-height: 60px;
+
+  text-decoration: none;
+  color: $main-text-color;
+
+  &:hover {
+    color: $active-color;
```

Comment:
Verify that the variable $active-color is defined in your variables.scss file and that it accurately reflects the color intended for active/hover states in your project.